introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
<p>These files are currently downloaded from public s3 buckets of JumpStart.</p><p>I understand that it would be best to do so. But if it is not a critical thing, can we do it in the next revision?</p>